lib/decimal.ex

```diff
@@ -447,6 +447,63 @@ defmodule Decimal do
     end
   end
 
+  @doc """
+  Compares two numbers numerically and returns `true` if they are equal,
+  otherwise `false`. If one of the operands is a quiet NaN this operation 
+  will always return `false`.
+
+  ## Examples
+
+      iex> Decimal.eq?("1.0", 1)
+      true
+
+      iex> Decimal.eq?(1, -1)
+      false
+
+  """
```

Comment:
```suggestion
  """
  doc_since("1.8.0")
```

(`doc_since` is our little wrapper around `@doc since: "..."` for older Elixir versions)